app/controllers/oral_history_access_requests_controller.rb

```diff
@@ -0,0 +1,49 @@
+# PUBLIC FACING
+# Staff-facing actions are in app/controllers/admin/oral_history_access_requests_controller.rb
+class OralHistoryAccessRequestsController < ApplicationController
+
+  before_action :set_work
+
+  # GET /works/4j03d097t/request_oral_history_access
+  def new
+    @oral_history_access_request = cls.new
+  end
+
+  # POST "/request_oral_history_access"
+  def create
+    @oral_history_access_request = cls.new(oral_history_access_request_params)
+    @oral_history_access_request.work = @work
+    if @oral_history_access_request.save
+      redirect_to work_path(@work.friendlier_id), notice: 'Your request has been logged.'
+    else
+     render :new
+    end
+  end
+
+private
+  def cls
+    Admin::OralHistoryAccessRequest
+  end
```

Comment:
What's the point of putting this in a method? Why not just say `Admin::OralHistoryAccessRequest.new` above, instead of the layer of indirection? Actual question, you can have an answer! Curious where you saw this pattern or whatever.

Reply:
It was just sheer irritation at having such a bulky identifier for the class. Maybe this wasn't such a great idea :)